workflows/taxprofiler.nf

```diff
@@ -172,12 +175,20 @@ workflow TAXPROFILER {
                                     db: it[3]
                             }
 
+    ch_input_for_metaphlan3 = ch_input_for_profiling.metaphlan3
+                            .dump(tag: "input_metaphlan3")
+                            .multiMap {
+                                it ->
+                                    reads: [it[0] + it[2], it[1][0]]
```

Comment:
Is there a particular reason for `it[1][0]`? Why not just `it[1]`? Or is this to account for unmerged PE reads?

Reply:
For some reason, this appeared as a list of list in the channel. Since it's just a draft I didn't want to spend time to figure it out exactly. So this is just a quick fix.